File: cpp/bc1131.cpp
#include <iostream>

using namespace std;

int main()
{
	int vIn, vGre, epa, nuG;
	
	vIn = vGre = epa = nuG = 0;
	
	while(true)
	{
		int fVal, sVal, opc;
		
		cin >> fVal >> sVal;
		
		if(fVal == sVal) epa++;
		else if(fVal > sVal) vIn++;
		else vGre++;
		
		nuG++;
		
		cout << "Novo grenal (1-sim 2-nao)" << endl;
		cin >> opc;
		
		if(opc == 2) break;
	}
	
	cout << nuG << " grenais" << endl;
	cout << "Inter:" << vIn << "\nGremio:" << vGre << "\nEmpates:" << epa << endl;
	
	string vit = "";
	
	if(vIn == vGre) vit = "Nao houve vencedor";
	else if(vIn > vGre) vit = "Inter venceu mais";
	else vit = "Gremio venceu mais";
	
	cout << vit << endl;
}

File: python/bc1131.py
vIn = 0
vGre = 0
epa = 0
nuG = 0

while True:
    cVal = input().split(" ")

    fVal = int(cVal[0])
    sVal = int(cVal[1])

    if fVal == sVal: epa += 1
    elif fVal > sVal: vIn += 1
    else: vGre += 1

    nuG += 1

    print("Novo grenal (1-sim 2-nao)")
    opc = int(input())

    if opc == 2: break

print(f"{nuG} grenais")
print(f"Inter:{vIn}\nGremio:{vGre}\nEmpates:{epa}")

vit = ""

if vIn == vGre: vit = "Nao houve vencedor"
elif vIn > vGre: vit = "Inter venceu mais"
else: vit = "Gremio venceu mais"

print(vit)